Lib/multiprocessing/forkserver.py

```diff
@@ -174,6 +174,8 @@ def main(listener_fd, alive_r, preload, main_path=None, sys_path=None):
                 spawn.import_main_path(main_path)
             finally:
                 del process.current_process()._inheriting
+        if sys_path is not None:
+            sys.path[:] = sys_path
```

Comment:
Should not it be set before loading the `__main__` module?

Reply:
I glanced at that code and don't think it is actually importing anything? t'd be safer to put it there regardless, I'll move it.

Reply:
It runs the `__main__` code which can import anything.